Changelog.md

```diff
@@ -1,3 +1,7 @@
+# v0.8.19 2018-11-06
+
+* Fix bug when mutating a module that includes Memoizable
```

Comment:
Lets also mention we dropped Ruby < 2.5 support.

Reply:
@mbj Done.